2-kubernetes/README.md

```diff
@@ -106,7 +114,7 @@ With:
   
 | Parameter | Description |
 | --- | --- | 
-| AGENT_SIZE | The size of K8s's agent VM. For this workshop use `Standard_NC6` if you want GPUs or `Standard_D2_v2` for CPU only. |
+| AGENT_SIZE | The size of K8s's agent VM. `Standard_D2_v2` is enough for this workshop. |
```

Comment:
shouldn't we mention 
> use `Standard_NC6` if you want GPUs

Reply:
I want to avoid people creating GPU clusters for now, unless they really know what they are doing.

Reply:
ok